### PR TITLE
fixed staging test execution to work with device-connect capable client

### PR DESCRIPTION
**Pull request by mzedel**
Changelog: None
Signed-off-by: Manuel Zedel <manuel.zedel@northern.tech>

### CODE REVIEW

**Comment by mzedel** [comment]
https://gitlab.com/Northern.tech/Mender/gui/-/pipelines/241674460 <- example pipeline with successfully executed job against staging...

File: tests/e2e_tests/dockerClient/mender.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "InventoryPollIntervalSeconds": 5,
+  "RetryPollIntervalSeconds": 30,
+  "ServerURL": "https://docker.mender.io/",
+  "UpdatePollIntervalSeconds": 5,
+  "ServerCertificate": "/usr/share/doc/mender-client/examples/demo.crt"
+}
```
Comment (merlin-northern) [inline tests/e2e_tests/dockerClient/mender.json:7]: where is that file used?

File: tests/e2e_tests/cypress/plugins/index.js [hunk 2]
```diff
@@ -5,37 +5,81 @@ const fs = require('fs');
 
 const { addMatchImageSnapshotPlugin } = require('cypress-image-snapshot/plugin');
 
+const updateConfigFileWithUrl = (fileName, serverUrl = 'https://docker.mender.io', token = '', projectRoot) => {
+  const connectConfigFile = fs.readFileSync(`dockerClient/${fileName}.json`);
```
Comment (merlin-northern) [inline tests/e2e_tests/cypress/plugins/index.js:9]: ah ok here. sorry I see it.

File: tests/e2e_tests/cypress/plugins/index.js [hunk 3]
```diff
@@ -5,37 +5,81 @@ const fs = require('fs');
...
+      updateConfigFileWithUrl('mender-connect', baseUrl, token, config.projectRoot);
+      let args = [
+        'run',
+        '-v',
+        `${config.projectRoot}/hosted.pem:/certs/hosted.pem`,
+        '-v',
+        `${config.projectRoot}/dockerClient/mender-test.json:/etc/mender/mender.conf`,
+        '-v',
+        `${config.projectRoot}/dockerClient/mender-connect-test.json:/etc/mender/mender-connect.conf`,
+        'mendersoftware/mender-client-docker:connect'
```
Comment (merlin-northern) [inline tests/e2e_tests/cypress/plugins/index.js:66]: so, `:connect` will always exists, right? maybe lets make some variable of it if not a parameter.

Comment (mzedel) [reply tests/e2e_tests/cypress/plugins/index.js:66]: it's built in an earlier job for that specific purpose (running it in the e2e tests to test the OS setup + in these tests against staging)

**Review by merlin-northern — APPROVED** [review]
lgtm (I am not an expert on cypress framework)
nice!

**Pull request merged**